### PR TITLE
Release v4.4.23

### PR DESCRIPTION
**Pull request by dvaldivia**
Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Review by Alevsk — APPROVED** [review]

File: docs/crd.adoc [hunk 1]
```diff
@@ -4,10 +4,10 @@
 [id="{p}-api-reference"]
 == API Reference
 
-:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2022-05-26T05-48-41Z]
+:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2022-06-03T01-40-53Z]
```
Comment (harshavardhana) [inline docs/crd.adoc:7]:
```suggestion
:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2022-06-07T00-33-41Z]
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: docs/crd.adoc (hunk 1 above)
Comment (dvaldivia) [reply docs/crd.adoc]: done

**Review by harshavardhana — DISMISSED** [review]

**Review by Alevsk — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

File: manifests/minio-operator-rhmp.clusterserviceversion.yaml [hunk 2]
```diff
@@ -685,7 +685,7 @@ spec:
                     env:
                       - name: CONSOLE_OPERATOR_MODE
                         value: "on"
-                    image: quay.io/minio/console@sha256:dd181fd701b6a7e7787d7f5b63653367aca2bcac151a5c204152a44a7604c092
+                    image: quay.io/minio/console@
```
Comment (harshavardhana) [inline manifests/minio-operator-rhmp.clusterserviceversion.yaml:688]: is this okay? @dvaldivia

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: manifests/minio-operator-rhmp.clusterserviceversion.yaml (hunk 2 above)
Comment (dvaldivia) [reply manifests/minio-operator-rhmp.clusterserviceversion.yaml:688]: @cniackz this doesn't seem right, right?

Comment (dvaldivia) [reply manifests/minio-operator-rhmp.clusterserviceversion.yaml]: fixed @harshavardhana

**Review by harshavardhana — APPROVED** [review]

**Review by cniackz — APPROVED** [review]
LGTM all tests are passing here.

**Review by Alevsk — APPROVED** [review]

**Pull request merged**